git_machete/git_operations.py

```diff
@@ -774,15 +774,23 @@ def is_ancestor_or_equal(
             earlier_revision: AnyRevision,
             later_revision: AnyRevision,
     ) -> bool:
+        earlier_hash = self.get_commit_hash_by_revision(earlier_revision)
+        return self.get_merge_base(earlier_revision, later_revision) == earlier_hash
+
+    def get_merge_base(
```

Comment:
@PawelLipski, it is confusing to have almost the same method name as the one method above -> `__get_merge_base` and call it inside this function. Maybe change the name of one of the functions (__get_merge_base?) or remove one and consolidate the code in the other, wdyt?